src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/service/SubmissionServiceImpl.java

```diff
@@ -164,8 +165,8 @@ private Document toDocument(String courtLevel, String courtClass, DocumentProper
 
     private BigDecimal getStatutoryFeeAmount(GenerateUrlRequest request) {
 
-        request.getClientApplication().setType(request.getClientApplication().getType());
-        ServiceFees fee = efilingLookupService.getServiceFee(request.getClientApplication().getType());
+        request.getClientApplication().setType(SubmissionConstants.STATUTORY_FEE_TYPE);
+        ServiceFees fee = efilingLookupService.getServiceFee(SubmissionConstants.STATUTORY_FEE_TYPE);
```

Comment:
is that not the submission fees ? Stat fees are per documents right ?

Reply:
Yeah you are right. I was just looking at the method name :P I will update that as well!